administrator/language/en-GB/com_scheduler.ini

```
@@ -0,0 +1,143 @@
+; Joomla! Project
+; (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
+; GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+COM_SCHEDULER="Scheduled Tasks"
+COM_SCHEDULER_CONFIGURATION="Scheduled Tasks Manager Configuration"
+COM_SCHEDULER_CONFIG_FIELDSET_LAZY_SCHEDULER_DESC="Configure how site visits trigger Scheduled Tasks."
+COM_SCHEDULER_CONFIG_FIELDSET_LAZY_SCHEDULER_LABEL="Lazy Scheduler"
+COM_SCHEDULER_CONFIG_GENERATE_WEBCRON_KEY_DESC="The webcron needs a protection key before it is functional. Saving this configuration will autogenerate the key."
+COM_SCHEDULER_CONFIG_GLOBAL_WEBCRON_KEY_LABEL="Global Key"
+COM_SCHEDULER_CONFIG_GLOBAL_WEBCRON_LINK_DESC="By default, requesting this base link will only run tasks due for execution. To execute a specific task, use the task's ID as a query parameter appended to the URL: <code>BASE_URL&task_id=&lttask's id&gt</code>"
+COM_SCHEDULER_CONFIG_GLOBAL_WEBCRON_LINK_LABEL="Webcron Link (Base)"
+COM_SCHEDULER_CONFIG_HASH_PROTECTION_DESC="If enabled, tasks will only be triggered when URLs have the scheduler hash as a parameter."
+COM_SCHEDULER_CONFIG_LAZY_SCHEDULER_ENABLED_DESC="If disabled, scheduled tasks will not be triggered by visitors on the site.<br>Recommended if triggering with native cron."
+COM_SCHEDULER_CONFIG_LAZY_SCHEDULER_ENABLED_LABEL="Lazy Scheduler"
+COM_SCHEDULER_CONFIG_LAZY_SCHEDULER_INTERVAL_DESC="Interval between scheduler trigger requests from the client."
+COM_SCHEDULER_CONFIG_LAZY_SCHEDULER_INTERVAL_LABEL="Request Interval"
+COM_SCHEDULER_CONFIG_RESET_WEBCRON_KEY_LABEL="Reset Access Key"
+COM_SCHEDULER_CONFIG_TASKS_FIELDSET_LABEL="Configure Tasks"
+COM_SCHEDULER_CONFIG_TASK_TIMEOUT_LABEL="Task Timeout (seconds)"
+COM_SCHEDULER_CONFIG_WEBCRON_DESC="Trigger and manage task execution with an external service."
+COM_SCHEDULER_CONFIG_WEBCRON_ENABLED_LABEL="Web Cron"
+COM_SCHEDULER_CONFIG_WEBCRON_LABEL="Web Cron"
+COM_SCHEDULER_CONFIG_WEBCRON_LINK_COPY_DESC="Copy the link to your clipboard."
+COM_SCHEDULER_CONFIG_WEBCRON_LINK_COPY_FAIL="Could not copy link!"
+COM_SCHEDULER_CONFIG_WEBCRON_LINK_COPY_SUCCESS="Link copied!"
+COM_SCHEDULER_DASHBOARD_TITLE="Scheduled Tasks Manager"
+COM_SCHEDULER_DESCRIPTION_TASK_PRIORITY="⚠ This is an advanced option. Higher priority tasks can potentially block out lower priority tasks."
```

```suggestion
COM_SCHEDULER_DESCRIPTION_TASK_PRIORITY="This is an advanced option. Higher priority tasks can potentially block lower priority tasks."
```

best not to use emoji in language files

---

Thanks 😅